### DISCUSSION TITLE
Error: The query requires a COLLECTION_GROUP_ASC index for collection Hostels and field rollNumber.

### DISCUSSION THREAD

**Discussion by Ani-MBC** [AntlerVC/firetable#831] [Support / Q&A] [answered]
**Describe the bug**
I tried to do a filter on the table, which was created by importing csv. filter parameter was space " ".
It could be a problem with my data though!

Adding info from description:
You can create it here: https://console.firebase.google.com/v1/r/project/lmtsom-e9669/firestore/indexes?create_exemption=ClRwcm9qZWN0cy9sbXRzb20tZTk2NjkvZGF0YWJhc2VzLyhkZWZhdWx0KS9jb2xsZWN0aW9uR3JvdXBzL0hvc3RlbHMvZmllbGRzL3JvbGxOdW1iZXIQAhoOCgpyb2xsTnVtYmVyEAE

**Comment by notsidney** [comment, marked as answer]
Hi, you might be sorting another column. Firestore requires you to create an index if you’re sorting by one column and filtering by another. You can either unsort the column or click the link in the snackbar to enable that sort + filter.
